gramps/gen/proxy/filter.py

```diff
@@ -24,6 +24,12 @@
 """
 Proxy class for the Gramps databases. Apply filter
 """
+# -------------------------------------------------------------------------
+#
+# Standard python modules
+#
+# -------------------------------------------------------------------------
+from __future__ import annotations
```

Comment:
I might need to add this to all of the rules that I added typing hints too. Do we support a version of Python that needs this line?

Reply:
We support python 3.9 and above.  If you are using a feature such as the vertical bar for unions, you will need the `__future__` import.

Reply:
Yes, there was one: #1918